src/server/game/Battlegrounds/Zones/BattlegroundAV.cpp

```diff
@@ -493,11 +493,16 @@ void BattlegroundAV::RemovePlayer(Player* player, ObjectGuid /*guid*/, uint32 /*
 
 void BattlegroundAV::HandleAreaTrigger(Player* player, uint32 trigger, bool entered)
 {
-    if (GetStatus() != STATUS_IN_PROGRESS)
-        return;
-
     switch (trigger)
     {
+        case 6633: // Horde Start
+        case 6632: // Alliance Start
+            if (GetStatus() == STATUS_WAIT_JOIN && entered)
```

Comment:
Yes, this areatrigger is right infront of the gate (iirc the position of the gate on retail is different than on trinitycore)